apps/back-office/src/app/(dashboard)/settings/_components/SettingsSidebar.tsx

```
@@ -0,0 +1,228 @@
+
+export default function SettingsSidebar() {
+  const pathname = usePathname();
+  const [expandedMenus, setExpandedMenus] = useState<string[]>([]);
+
+  useState(() => {
+    let foundParentLabel: string | null = null;
+    Object.values(menuItems).forEach((section) => {
+      section.forEach((item: MenuItem) => {
+        if (
+          item.children &&
+          item.children.some((child: MenuItem) => child.href === pathname)
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)